nixos/modules/config/swap.nix

```diff
@@ -252,6 +252,7 @@ in
           let realDevice' = escapeSystemdPath sw.realDevice;
           in nameValuePair "mkswap-${sw.deviceName}"
           { description = "Initialisation of swap device ${sw.device}";
+            after = [ "systemd-modules-load.service" ];
```

Comment:
Please add a short comment based on your description here so that people in the future easily know why we have this.

Reply:
Done!